rfcs/experimental/flow_graph_node_sets/README.md

```diff
@@ -0,0 +1,262 @@
+    function_node squarer(g, unlimited, [](const int&) { return v * v; });
+    function_node cuber(g, unlimited, [](const int& v) { return v * v * v; });
+
+    make_edge(input, doubler);
+    make_edge(input, squarer);
+    make_edge(input, cuber);
+```
+
+To reduce verbosity and improve readability, additional functions were added to the flow graph API
+to simpify these cases. A shorter and more readable implementation of the previous code snippet that
+uses these extensions is shown below:
+
+```cpp
+    function_node doubler(g, unlimited, [](const int& v) { return 2 * v; });
+    function_node squarer(g, unlimited, [](const int&) { return v * v; });
+    function_node cuber(g, unlimited, [](const int& v) { return v * v * v; });
+
+    broadcast_node input(precedes(doubler, squarer, cuber));
+```
+
+## Proposal
+
+Several helper functions and additional constructors have been added as experimental features.
+These additions simplify expressing dependencies between nodes when building oneTBB flow graphs.
+This experimental feature is described in more detail in the
+[Developer Guide](https://www.intel.com/content/www/us/en/docs/onetbb/developer-guide-api-reference/2022-0/helper-functions-for-expressing-graphs.html)
+
+There are four main parts to the extension:
+
+- `make_node_set` function template
+- `make_edges` function template
+- two additional constructors for each flow graph node
+- `follows` and `precedes` function templates
+
+### Node Sets
+
+In this proposal, `node_set` is an exposition-only name for the type returned from the
+`make_node_set` function. The `make_node_set` function template creates a set of nodes that
+can be passed as arguments to the `make_edges`, `follows` and `precedes` functions that are
+described in later sections. All nodes in a `node_set` should be from the same graph. Including
+nodes from more than one graph in a `node_set` results in undefined behavior.
+
+```cpp
+    // node_set is an exposition-only name for the type returned from make_node_set function
+
+    template <typename Node, typename... Nodes>
+    /*unspecified*/ make_node_set( Node& node, Nodes&... nodes );
+```
+
+### The `make_edges` function template
+
+The make_edges function template creates edges between a single node and each node in a set of nodes.
+
+There are two ways to connect nodes in a set and a single node using make_edges:
+
+<img src="make_edges.png">
+
+The order of the arguments determines the predecessor / successor relationship. The node or nodes represented
+by the left argument becomes the predecessor or predecessors of the node or nodes represented by the right
+argument.
+
+### Additional Constructors for Flow Graph nodes
+
+Each flow graph node has been extended with two new constructors, one that receives a *follows* 
+set of nodes `[1]` and another that receives a *precedes* set of nodes `[2]`. These node sets are 
+created using the `precedes` and `follows` helper functions described in the next section.
```

Comment:
There is some semantical/naming nuance that may need to be explained. The names `follows` and `precedes` do not describe the node set of their arguments, referring instead to a to-be-constructed node and describing its relation to the node set. In other words, `precedes` creates a set of "following" nodes (successors). and `follows` creates a set of "preceding" nodes (predecessors).

It might be yet another area where some feedback would be helpful. If the names do not make the usage semantics clear, the goal of simplifying graph construction will not be achieved in full.

Reply:
Will add.